File: 2.0/java/keywords/isKeywordAvailable.java
import com.callfire.api.client.CallfireClient;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api_login", "api_password");
        Boolean available = client.keywordsApi().isAvailable("FIRE");
    }
}

File: 2.0/php/keywords/isKeywordAvailable.php
<?php

class ApiClientSample {

    public static function main() {
        $client = \CallFire\Api\DocumentedClient::createClient("login", "password");
        $request = $client->isKeywordAvailable();
        $request->getOperationConfig()->setPathParameters(array("keyword" => "SUN"));
        $result = $client->request($request);
        $json = json_decode($result->getBody());
    }
}

ApiClientSample::main();
